Duplicati/Server/webroot/ngax/scripts/directives/parseFilterType.js

```diff
@@ -31,23 +32,23 @@ backupApp.directive('parseFilterContent', function(AppUtils, SystemInfo) {
         restrict: 'A',
         require: ['ngModel'],
         link: function(scope, element, attr, ctrl) {
-
-            var type = null;
-
             ctrl[0].$parsers.push(function(txt) {
                 var dirsep = SystemInfo.state.DirectorySeparator || '/';
-                return AppUtils.buildFilter(type, txt, dirsep);
+                // Store type in scope for parseFilterType directive
```

Comment:
Should this comment not be 'Store content in scope' instead of 'store type in scope' ?